src/GraphQL/Authorization/AllowAnonymousAttribute.cs

```diff
@@ -0,0 +1,24 @@
+using GraphQL.Types;
+using GraphQL.Utilities;
+
+namespace GraphQL;
+
+/// <summary>
+/// Attribute to typically indicate that anonymous access should be allowed to a field of a graph type
+/// requiring authorization, providing that no other fields were selected.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Method | AttributeTargets.Field)]
```

Comment:
Someday in the future this list will expand. I came across some unusual cases when the non-standard logic of processing input parameters was required.

Reply:
No problem; it will be a non-breaking change at that time.